clients/geth/specular/data/base_genesis.json

```diff
@@ -13,15 +13,44 @@
   "difficulty": "1",
   "gasLimit": "8000000",
   "extradata": "",
-  "alloc": {
-    "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266": {
+  "preDeploy": [
+    {
+      "address": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
       "balance": "10000000000000000000"
     },
-    "0x70997970c51812dc3a010c7d01b50e0d17dc79c8": {
+    {
+      "address": "0x70997970c51812dc3a010c7d01b50e0d17dc79c8",
       "balance": "10000000000000000000"
     },
-    "0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc": {
+    {
+      "address": "0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc",
       "balance": "10000000000000000000"
+    },
+    {
+      "address": "0x0000000000000000000000000000000000000020",
+      "contract": "Faucet",
+      "balance": "100000000000000000000",
+      "storage": {
+        "_owner": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
```

Comment:
although not sure if we want to implement in this pr (maybe a separate pr is better), but it would be helpful if we allow some "macro" here, e.g. use `<SEQUENCER-ADDR>` here instead of hard encoding can be much more easier for configuration.